pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -0,0 +1,195 @@
+{ lib
+, fetchurl
+, fetchpatch
+, nixosTests
+, python3
+, ghostscript
+, imagemagick
+, jbig2enc
+, optipng
+, pngquant
+, qpdf
+, tesseract4
+, unpaper
+, liberation_ttf
+}:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_3;
+
+      # Incompatible with aioredis 2
+      aioredis = super.aioredis.overridePythonAttrs (oldAttrs: rec {
+        version = "1.3.1";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "0fi7jd5hlx8cnv1m97kv9hc4ih4l8v15wzkqwsp73is4n0qazy0m";
+        };
+      });
+    };
+  };
+
+  path = lib.makeBinPath [ ghostscript imagemagick jbig2enc optipng pngquant qpdf tesseract4 unpaper ];
+in
+py.pkgs.pythonPackages.buildPythonApplication rec {
+  pname = "paperless-ngx";
+  version = "1.6.0";
+
+  src = fetchurl {
+    url = "https://github.com/paperless-ngx/paperless-ngx/releases/download/ngx-${version}/${pname}-${version}.tar.xz";
+    sha256 = "07mrxbwahkm00n9nvssd6d13p80w333g84cd38bzp0l34nzim5zl";
+  };
+
+  format = "other";
+
+  # Make bind address configurable
+  postPatch = ''
+    substituteInPlace gunicorn.conf.py --replace "bind = " "# bind = "
```

Comment:
should be:
```
substituteInPlace gunicorn.conf.py --replace "bind = f'0.0.0.0:{os.getenv(\"PAPERLESS_PORT\", 8000)}'" ""
```

according to https://github.com/paperless-ngx/paperless-ngx/blob/5ead8de0bb5d4fda08404aacc65d6d3405348d53/gunicorn.conf.py#L3

Reply:
I guess we can get away with just commenting out.